frontend/src/lib/templates/page/parts/CandidateNav.svelte

```diff
@@ -62,7 +63,12 @@ A template part that outputs the navigation menu for the Candidate App for use i
       href={$getRoute(Route.CandAppPreview)}
       icon="previewProfile"
       text={$t('candidateApp.preview.title')} />
-    <NavItem href={$getRoute(Route.CandAppHelp)} icon="help" text={$t('candidateApp.help.title')} />
+    {#if $customization.candidateAppFAQ?.length}
```

Comment:
Help is currently just the FAQ. Perhaps we can add a common help intro in the future and always show the item.